libs/logging-utils/src/lib.rs

```diff
@@ -57,23 +59,45 @@ pub fn convert_vx_log_to_cdf(
         })?;
     let input_path = PathBuf::from(input_path);
     let output_path = PathBuf::from(output_path);
+
+    let log_as_current_role: ThreadsafeFunction<LogData, ErrorStrategy::Fatal> =
+        log_as_current_role.create_threadsafe_function(0, |ctx| {
```

Comment:
Just ran into an issue while testing - since the TsFn isn't bound to the logger's `this` context, calling it crashes when using a real logger (the mock logger doesn't rely on `this`).

Not sure yet how to do this with `napi-rs`, but thinking we might need to create a ref to the logger and then deref it in the TsFn handler so we can invoke the callback with the appropriate `this` - just couldn't figure out if that's possible with `napi-rs`' built-in TsFn abstraction.

Might have time to dig in more later tomorrow, but LMK if you have enough time/context to take a look today

Reply:
Come to think of it, maybe it's easier for the JS code to pass in a wrapper log function instead of a logger object so we don't have to worry about this on the Rust side 🤔

Reply:
Yeah, I was thinking the same. I'll make a PR for it.

Reply:
https://github.com/votingworks/vxsuite/pull/6333